mods/ra/maps/monster-tank-madness/weapons.yaml

```diff
@@ -7,28 +7,12 @@ TurretGun:
 		Blockable: false
 
 SuperTankPrimary:
-	Inherits: ^Cannon
-	ROF: 70
+	Inherits: 120mm
+	ReloadDelay: 70
 	Range: 4c768
 	Report: turret1.aud
-	Burst: 2
-	InvalidTargets: Air, Infantry
-	Projectile: Bullet
-		Speed: 682
-		Image: 120MM
 	Warhead@1Dam: SpreadDamage
-		Spread: 128
-		Damage: 50
+		Damage: 500
 		InvalidTargets: Air, Infantry
 		Versus:
 			None: 20
```

Comment:
Minor nit: Since InvalidTargets makes this warhead completely useless vs. infantry, this custom Versus value is fully redundant (there aren't any non-infantry actors with this armor type, afaik).